helm-chart/csi-driver/templates/linode-block-storage-retain.yaml

```diff
@@ -7,10 +7,13 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
 {{- end }}
-{{- if .Values.volumeTags }}
 parameters:
+{{- if .Values.volumeTags }}
   linodebs.csi.linode.com/volumeTags: {{ join "," .Values.volumeTags }}
 {{- end}}
+{{- if .Values.volumeEncryption }}
+  linodebs.csi.linode.com/encrypted: "true"
+{{- end}}
```

Comment:
We don't want to render parameters if both volume tags and volumeEncryption is not passed in which is highly likely since they are both optional

Reply:
Sure, I'll change that